src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -209,6 +210,8 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         get(API_ROOT + RoutePath.BSP_KIT_REGISTERED, new BSPKitRegisteredRoute(), new JsonTransformer());
         if(!cfg.getBoolean("ui.production")){
             get(API_ROOT + RoutePath.DUMMY_ENDPOINT, new MercuryDummyKitEndpoint(), new JsonTransformer());
+            get(API_ROOT + RoutePath.CLINICAL_KIT_ENDPOINT, new ClinicalKitsRoute(notificationUtil), new JsonTransformer());
```

Comment:
this is a real endpoint we need in prod (later on, so I would already move it to where the other gp route is